.github/workflows/fmt.yml

```
@@ -27,4 +31,5 @@ jobs:
           toolchain: stable
           override: true
           components: clippy
-      - run: cargo clippy --tests --all-targets --all-features -- --D warnings
+      - name: Run test with ${{ matrix.features}}
+        run: cargo clippy --tests --all-targets --features=${{ matrix.features }} -- --D warnings --no-deps
```

```suggestion
        run: cargo clippy --tests --all-targets --features=${{ matrix.features }} --no-deps -- --D warnings
```
I might be wrong, but think `--no-deps` goes on the clippy side